src/buspirate.c

```diff
@@ -377,6 +377,21 @@ buspirate_parseextparms(const PROGRAMMER *pgm, const LISTID extparms) {
 			continue;
 		}
 
+    if (str_eq(extended_param, "help")) {
+      char *prg = (char *)ldata(lfirst(pgm->id));
+      msg_error("%s -c %s extended options:\n", progname, prg);
+      msg_error("  -xreset=cs,aux,aux2         Override default reset pin\n");
+			msg_error("  -xspifreq=<0..7>            Set binary SPI mode speed\n");
+			msg_error("  -xrawfreq=<0..3>            Set \"raw-wire\" SPI mode speed\n");
```

Comment:
Mixture of tabs and space for indentation in new code?

Reply:
Ah, I see. It's my text editor that didn't automatically switch to tabs.